packages/assets-controllers/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - **BREAKING:** Add peer dependency on `@metamask/phishing-controller` ^12.5.0 ([#5598](https://github.com/MetaMask/core/pull/5598))
+- Bump `@metamask/preferences-controller` to `^18.1.0` ([#5870](https://github.com/MetaMask/core/pull/5870))
```

Comment:
Since this is a bump to a dev dependency, should we include it in our changelog?

Reply:
IIRC we only include (major) bumps to our peer dependencies on controllers.

Reply:
This bump was again done by release script, I can revert it though.

Reply:
@jpuri yes please

Reply:
CI is complaining to update asset-controller dependency